train.py

```diff
@@ -1,18 +1,33 @@
+import csv
 import random
 
 from autowebcompat import network, utils
 
 labels = utils.read_labels()
 
+def update_all_image_names(images):
+    with open('labels.csv') as csvfile:
```

Comment:
You are re-reading the file that has already been read with utils.read_labels()  just a few lines earlier.